Gemfile

```diff
@@ -41,6 +41,9 @@ gem 'webpacker', '~> 4.x'
 gem 'google-api-client', '0.32.1'
 gem 'googleauth'
 
+gem 'ed25519', '~> 1.2'
+gem 'bcrypt_pbkdf', '~> 1'
```

Comment:
Can I check why these 2 gems are needed here?

Reply:
the aren't strictly needed, they just allow ed25519 keys to be used as well as RSA keys for deploying with Capistrano.

 i don't know much about it, but ed25519 are somehow considered "better", and e.g. github's first suggestion in setting up keys is to use ed25519.

So i can just remove these gems and set up RSA keys instead, but other people might have to go through switching them around in future i guess? happy to leave or remove

Reply:
makes sense, let's keep those for future proof